utils/python/pipenvutils.go

```diff
@@ -0,0 +1,83 @@
+package python
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"strings"
+)
+
+func getPipenvEnvironmentString(venvDir string) string {
+	if venvDir != "" {
+		return fmt.Sprintf("WORKON_HOME=%s", venvDir)
+	}
+	return ""
+}
+
+// Execute "pipenv --venv" to get the pipenv virtual env path
+func GetPipenvVenv(venvDir string) (string, error) {
+	output, err := runPythonCommand("pipenv", []string{"--venv"}, getPipenvEnvironmentString(venvDir))
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSuffix(string(output), "\n"), err
+}
+
+// Executes the pipenv graph and returns a dependency map of all the installed pip packages in the current environment to and another list of the top level dependencies
+func GetPipenvDependenciesList(venvDir string) (map[string]bool, error) {
+	packages, err := runPipenvGraph(venvDir)
+	if err != nil {
+		return nil, err
+	}
+
+	// Parse the result.
+	return parseDependenciesToList(packages)
+}
+
+// Executes the pipenv install and pipenv graph
```

Comment:
// Executes the pipenv install and pipenv graph
-->
// Executes pipenv install and pipenv graph.